kpops/cli/main.py

```diff
@@ -205,6 +206,23 @@ def generate(
         None, help="Path to file the yaml pipeline should be saved to"
     ),
     verbose: bool = typer.Option(False, help="Enable verbose printing"),
+    template: bool = typer.Option(False, help="Run helm template"),
+    steps: Optional[str] = PIPELINE_STEPS,
```

Comment:
`steps` is a generic filtering mechanism of KPOps and can be used for generating the `pipeline.yaml` and, of course, templating.